sympy/printing/fcode.py

```diff
@@ -129,6 +131,32 @@ def _lead(self):
         else:
             raise ValueError("Unknown source format: %s" % self._settings['source_format'])
 
+    def _print_Symbol(self, expr):
+        if self._settings['name_mangling'] == True:
+            for sym in tuple(ordered(expr.atoms(Symbol))):
```

Comment:
`expr` is an instance of `Symbol`

Reply:
Yes, he is absolutely right (I missed this -- that's why multiple reviewers is a good thing). Yes, go ahead and simplify this logic as much as you can.